src/main/java/nl/hsac/fitnesse/fixture/util/JsonPathHelper.java

```diff
@@ -54,4 +51,12 @@ public Object getJsonPath(String json, String jsonPath) {
         }
         return result;
     }
+
+    public String updateJsonPathWithValue(String json, String jsonPath, String value) {
+        if(null != getJsonPath(json, jsonPath)) {
```

Comment:
That's intended behaviour for now (in my case, I don't need it and I think it would encourage to build fragile tests.. theoretically you could then get any arbitrary json and abuse the fixture to create something entirely different. Would not like to debug that). I didn't include an option to delete objects either :-)